boards/nucleo-wl55jc/Makefile.dep

```diff
@@ -1,7 +1,7 @@
 ifneq (,$(filter stdio_uart,$(USEMODULE)))
   FEATURES_REQUIRED += periph_lpuart
 endif
-ifneq (,$(filter netdev_default,$(USEMODULE)))
+ifneq (,$(filter sx126x%,$(USEMODULE)))
```

Comment:
why change this?

Reply:
I think it a mishap of rebasing my branch that was based on an earlier version of the branch adding support for the `sx126x` spi radio.